treeherder/perf/migrations/0020_add_test_display_names.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.6 on 2019-11-01 11:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0015_add_repository_tc_root_url'),
+        ('perf', '0019_remove_confirming_state'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='performancesignature',
+            name='suite_public_name',
+            field=models.CharField(max_length=30, null=True),
+        ),
+        migrations.AddField(
+            model_name='performancesignature',
+            name='test_public_name',
+            field=models.CharField(max_length=30, null=True),
+        ),
+        migrations.AlterUniqueTogether(
+            name='performancesignature',
+            unique_together={('repository', 'framework', 'platform', 'option_collection', 'suite', 'test', 'last_updated', 'extra_options'), ('repository', 'framework', 'platform', 'option_collection', 'suite_public_name', 'test_public_name', 'extra_options'), ('repository', 'framework', 'signature_hash')},
```

Comment:
Do we still actually need all of these unique_together combinations?

Reply:
We need these to ensure we won't have any ambiguities in the `Add test data` modal from Graphs view, when selecting tests from the same framework, same repo & same platform.

In time, we want to get stricter about the naming conventions of our tests. @davehunt could you confirm this?